stp_zmq/zstack.py

```diff
@@ -432,17 +437,14 @@ def getHa(self, name):
         return 0
 
     def _verifyAndAppend(self, msg, ident):
-        # if self.verify(msg, ident):
-        #     self.rxMsgs.append((msg[:-self.sigLen].decode(), ident))
-        # else:
-        #     logger.error('{} got error while '
-        #                  'verifying message {} from {}'
-        #                  .format(self, msg, ident))
         try:
+            self.checkMsgLenWithException(msg)
             decoded = msg.decode()
-        except UnicodeDecodeError as ex:
-            logger.error('{} got exception while decoding {} to utf-8: {}'
-                         .format(self, msg, ex))
+        except Exception as ex:
```

Comment:
Please do not catch generic `Exception`, catch specific exception for MsgLength check only